doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -89,24 +89,28 @@ The name of these tables depends on the configured data model (see the [Configur
 * Data model by entity type (`data_model=dm-by-entity-type`). For each entity, the notified/default FIWARE service path is concatenated to the notified entity type in order to compose the table name. The concatenation character is `_` (underscore). If the FIWARE service path is the root one (`/`) then only the entity type is concatenated.
 * Data model by entity database schema (`data_model=dm-by-entity-type-database-schema`). Same as Data model by entity type(`data_model=dm-by-entity-type`).
 * Data model by entity type database (`data_model=dm-by-entity-type-database`). Same as Data model by entity (`data_model=dm-by-entity-type`).
+* Data model by fixed entity type (`data_model=dm-by-fixed-entity-type`). For each entity, the notified entity type is used as a fixed table name.
+* Data model by fixed entity database schema (`data_model=dm-by-fixed-entity-type-database-schema`). Same as Data model by fixed entity type(`data_model=dm-by-fixed-entity-type`).
+* Data model by fixed entity type database (`data_model=dm-by-fixed-entity-type-database`). Same as Data model by fixed entity type (`data_model=dm-by-fixed-entity-type`).
```

Comment:
CNR entry mentioning the new (three?) datamodels should be added

Reply:
done in 85044b43ba265b4467e4607940e6570134575b4c